From c4c4d78141e67fece20bb6c8da1ad303df9ad74e Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.org>
Date: Wed, 5 Jun 2019 21:32:03 +0100
Subject: [PATCH 351/587] BCM270X_DT: Leave bulk channel in dma channel mask

The updated bcm2835-dma driver does not require the BULK channel
to be removed from the set of available channels, as provided by
dma-channel-mask.

Signed-off-by: Phil Elwell <phil@raspberrypi.org>
---
 arch/arm/boot/dts/bcm2708-rpi.dtsi | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm/boot/dts/bcm2708-rpi.dtsi b/arch/arm/boot/dts/bcm2708-rpi.dtsi
index 57958c642381..a267807f3d6f 100644
--- a/arch/arm/boot/dts/bcm2708-rpi.dtsi
+++ b/arch/arm/boot/dts/bcm2708-rpi.dtsi
@@ -124,10 +124,6 @@
 	};
 };
 
-&dma {
-	brcm,dma-channel-mask = <0x7f34>;
-};
-
 &hdmi {
 	power-domains = <&power RPI_POWER_DOMAIN_HDMI>;
 };
-- 
2.17.1

